run_release.py

```diff
@@ -252,7 +252,9 @@ def __init__(
         print(f"- Normalized release tag: {release_tag.normalized()}")
         print(f"- Git repo: {self.db['git_repo']}")
         print(f"- SSH username: {self.db['ssh_user']}")
-        print(f"- python.org API key: {self.db['auth_info']}")
+        user, key = self.db["auth_info"].split(":")
```

Comment:
Is there a reason to print it at all? We can remove the whole line if it's not useful.

Reply:
It's probably not needed -- do any @python/release-managers find this useful?

It was originally added when the script was created by @pablogsal in https://github.com/python/release-tools/pull/8.

I've only once needed to see it, to double check I was using the correct API key (I'd actually got the username wrong). And I could have added a temporary print on that occasion.